### PR TITLE
Master -> Main Rename

### PR DESCRIPTION
**Pull request by dannylamb**
Part of https://github.com/Islandora/documentation/issues/1595

### CODE REVIEW

File: config/install/views.view.iiif_manifest.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline config/install/views.view.iiif_manifest.yml]: Should this be proper case, as it is the displayed title?

File: config/install/views.view.members.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline config/install/views.view.members.yml]: Should this be proper case, as it is the displayed title?

File: config/install/views.view.openseadragon_media_evas.yml [hunk 3]
```diff
@@ -23,7 +23,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline config/install/views.view.openseadragon_media_evas.yml]: Should this be proper case, as it is the displayed title?

File: config/install/views.view.pdfjs_media_evas.yml [hunk 4]
```diff
@@ -23,7 +23,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline config/install/views.view.pdfjs_media_evas.yml]: Should this be proper case, as it is the displayed title?

File: modules/islandora_oaipmh/config/install/views.view.oai_pmh.yml [hunk 5]
```diff
@@ -19,7 +19,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline modules/islandora_oaipmh/config/install/views.view.oai_pmh.yml]: Should this be proper case, as it is the displayed title?

File: modules/islandora_search/config/install/views.view.solr_search_content.yml [hunk 6]
```diff
@@ -16,7 +16,7 @@ display:
   default:
     display_plugin: default
     id: default
-    display_title: Master
+    display_title: main
```
Comment (whikloj) [inline modules/islandora_search/config/install/views.view.solr_search_content.yml]: Should this be proper case, as it is the displayed title? 😌

**Comment by dannylamb** [comment]
:broom:

**Comment by dannylamb** [comment]
@Islandora/8-x-committers This one is ripe for merging ^_^

**Review by rosiel — APPROVED** [review]

**Pull request merged**